blockly/src/server/Server.java

```diff
@@ -1104,6 +1104,9 @@ private void performAction(String action) {
         }
         shootFireBall(Direction.fromString(firstArg));
       }
+      case "warte" -> {
+        waitDelta();
```

Comment:
`waitDelta` ist evtl zu kurz. Vlt. würde es mehr sind machen `Thread.sleep` zu verwenden?

Reply:
Dann bist du aber off-the-loop unterwegs. 

In den Kontext in den ich es verwenden will (Rätsel) passt das.
Das wird immer in einer while-Schleife abgefragt. 

Ich behalte es aber im Hinterkopf